js/app.js

```diff
@@ -0,0 +1,80 @@
+// 1. Verbal questions
+
+// a parameter is a variable that lives inside of a method
+// an argument is the value of after its been passed through the method
+// is it possible then to have your parameter and agrument be the same?
+// thats what it looks like in my Palindrome?
+
+// return is giving you the value of the function 
+// where console.log is just showing the output in the console
+
+// Other than returing a function value you can reuse it, pass it through an argument?
```

Comment:
you can capture the output of a function in a variable.  you can use a function that returns a value anywhere you could use that value 
for example: 
```
myArray[getArrayIndex()]
```